mcs/class/corlib/Makefile

```diff
@@ -37,11 +37,13 @@ RESX_RESOURCE_STRING = \
 	../../../external/corefx/src/System.Collections.NonGeneric/src/Resources/Strings.resx \
 	../../../external/corefx/src/System.ComponentModel.TypeConverter/src/Resources/Strings.resx \
 	../../../external/corefx/src/System.Buffers/src/Resources/Strings.resx \
+	../../../external/corefx/src/System.IO.FileSystem/src/Resources/Strings.resx \
```

Comment:
Why do we need them in mscorlib?

Reply:
Somebody manually edited the SR files instead of adding them to the `RESX_RESOURCE_STRING`, so I had to fix that.